import_shims/warn.py

```diff
@@ -39,6 +41,7 @@ def warn_deprecated_import(old_import, new_import):
     """
     Warn that a module is being imported from its old location.
     """
+    set_custom_attribute("deprecated_edx_platform_import", old_import)
```

Comment:
Two questions:
1. I set "value" to `old_import`, under the understanding that the deprecated import path is what we'd want to make visible in NewRelic. Is that right?
2. If one request uses, say, five different deprecated imports, will `deprecated_edx_platform_import` show five different values? Or will it only capture the last-used one?

Reply:
1. Sounds right to me.
2. Last value, so it will be like an onion.  This is fine and a simple start.  The alternative is to use a RequestCache value to track a list of old imports, and use that value to set the custom attribute.

Reply:
Great, thanks!